src/app/components/form/DualCurrencyField/range.tsx

```diff
@@ -0,0 +1,19 @@
+import { useTranslation } from "react-i18next";
```

Comment:
Not sure if this component makes sense. If there is a more elegant way to solve that please let me know 😅

Reply:
It looks ok to me. Maybe "RangeLabel" as the name? and it might be good to throw an exception or return null instead of an empty fragment if neither field is passed.